src/routers/MultiRouteRouter.js

```diff
@@ -46,7 +46,7 @@ class MultiRouteRouter extends ApplicationRouter<Array<Object>> {
 
     // return the best route for each destination
     return Promise.all(multiRoutes).then((val) => {
-      const bestRoutes = val.map(routes => ((routes.length > 0) ? routes[0] : null));
+      const bestRoutes = val.map(routes => (routes.length > 0 ? routes[0] : null));
```

Comment:
you don't need the `()` around the ternary

Reply:
when i autosave, it gets added?

Reply:
oh hm alright, @meganle i'm guessing it must be fine?